elide-core/src/test/java/example/IdFieldAnnotations.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package example;
+
+import com.yahoo.elide.annotation.Include;
+
+import java.util.Set;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+
+/**
+ * A test entity that has an ID field whose field name is not "id".
+ */
+@Include(rootLevel = true)
+@Entity
+public class IdFieldAnnotations {
```

Comment:
Two comments.  Probably only one of them needs to be addressed:
1. Can we remove all of the fields in this class that are needed for the test?
2. Instead of making a new test bean, could we use an existing one and just change the `@Id` field to a different name?

Reply:
Sure. That sounds good.